Task 1 - Shoe Object Modelling.txt

```diff
@@ -0,0 +1,5 @@
+Object Modeling Shoes:
```

Comment:
Move the file into resource folder
https://stackoverflow.com/questions/25786185/what-is-the-purpose-for-the-resource-folder-in-maven